dev_guide/limits.adoc

```diff
@@ -11,28 +11,84 @@ toc::[]
 
 == Overview
 
-A `*LimitRange*` object enumerates
-link:compute_resources.html[compute resource constraints] in a
-link:projects.html[project]. This allows you to specify the amount of resources
-that a pod or container can consume.
+A `*LimitRange*` object enumerates link:compute_resources.html[compute resource
+constraints] in a link:projects.html[project] at the pod and container level.
+This allows you to specify the amount of resources that a pod or container can
+consume.
 
-== Constraints
-
-A `*LimitRange*` object enumerates compute resource constraints at a pod and
-container scope in a project.
+== Setting Limit Range Constraints
 
 All resource create and modification requests are evaluated against each
-`*LimitRange*` object in the project.  If the resource violates any of the
-enumerated constraints, then the resource is rejected.  If the resource does
-not set an explicit value, and if the constraint supports a default value, then
-the default value is applied to the resource.
+`*LimitRange*` object in the project. If the resource violates any of the
+enumerated constraints, then the resource is rejected. If the resource does not
+set an explicit value, and if the constraint supports a default value, then the
+default value is applied to the resource.
+
+[[limit-range-def]]
+.Limit Range Object Definition
+====
+
+[source,json]
+----
+
+apiVersion: "v1"
+kind: "LimitRange"
+metadata: 
+  name: "limits" <1>
+spec: 
+  limits: 
+    - 
+      type: "Pod"
+      max: 
+        cpu: "2" <2>
+        memory: "1Gi" <3>
+      min: 
+        cpu: "200m" <4>
+        memory: "6Mi" <5>
+    - 
+      type: "Container"
+      max: 
+        cpu: "2" <6>
+        memory: "1Gi" <7>
+      min: 
+        cpu: "100m" <8>
+        memory: "4Mi" <9>
+      default: 
+        cpu: "300m" <10>
+        memory: "200Mi" <11>
+      defaultRequest: 
+        cpu: "200m" <12>
+        memory: "100Mi" <13>
+      maxLimitRequestRatio: 
+        cpu: "10" <14>
+----
+<1> The name of the limit range document.
+<2> The maximum amount of cpu that a pod can consume on a node across all
+containers.
+<3> The maximum amount of memory that a pod can consume on a node across all
+containers.
+<4> The minimum amount of cpu that a pod can consume on a node across all
```

Comment:
Maybe instead of "consume" we should say "request" since the majority of containers will consume no CPU when idle for example.